azure-pipelines.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           Arguments: 'build-test-app-V1'
           ProjectDirectory: '$(ClientSdkProjectDirectory)'
 
-      - bash: 'node tools/cli/runAppsWithE2ETests.js --envType=teamsV1 --targetClientSdkCheckpoint=1.11.0 --reportFileName=e2e-tests-report-V1'
+      - bash: 'node tools/cli/runAppsWithE2ETests.js --envType=teamsV1 --targetClientSdkCheckpoint=1.12.1 --reportFileName=e2e-tests-report-V1'
```

Comment:
I can see the v1 tests are failing (they haven't finished yet).

You are passing targetClientSdkCheckpoint=1.12.1, but I don't think that checkpoint exists in the code. Do you need to update `ClientSdkCheckpoint.v1` to point be '1.12.1' in cypress/utility/helper.ts?

Reply:
Yup was going to add this comment, thanks for beating me to it. We will need to make sure to update the endpoint for the E2E tests with 1.12.1 as well.

Reply:
Thank you for pointing this! I just merged a PR with changes in the host SDK